src/screens/ChatScreen/components/ReplyBox.js

```diff
@@ -185,6 +210,15 @@ const ReplyBox = ({ eva: { theme, style }, conversationId, cannedResponses }) =>
                 onPress={togglePrivateMode}
               />
             </View>
+            <View style={style.expandButton}>
+              <Icon
+                name={expandedInputButtonIcons()}
+                width={32}
```

Comment:
Seems like icons are big now. Please change icon size to 24 (Attachment, Private note and Expanded)

Reply:
<img width="399" alt="image" src="https://user-images.githubusercontent.com/64252451/168583867-709b5f33-2bde-4dd7-a623-59c529a08dee.png">

@muhsin-k Does the send button size ok?

Reply:
@iamsivin Keeps send button icon same

Reply:
@muhsin-k  I think the same size looks better. 

<img width="444" alt="image" src="https://user-images.githubusercontent.com/64252451/168635286-c772a357-3035-4d62-8519-66b96c45da9e.png">